src/OidcJwtHelper.php

```diff
@@ -94,17 +94,19 @@ public function verifyJwtClaims(string $issuer, ?object $claims, ?OidcTokens $to
     }
 
     // Get and remove nonce from session
-    $nonce = $this->sessionStorage->getNonce();
-    $this->sessionStorage->clearNonce();
+    if (true === $verifyNonce) {
+      $nonce = $this->sessionStorage->getNonce();
+      $this->sessionStorage->clearNonce();
+    }
 
     /* @noinspection PhpUndefinedVariableInspection */
     return ($claims->iss == $issuer)
         && (($claims->aud == $this->clientId) || (in_array($this->clientId, $claims->aud)))
-        && ($claims->nonce == $nonce)
+        && (false === $verifyNonce || $claims->nonce == $nonce)
```

Comment:
```suggestion
        && (!$verifyNonce || $claims->nonce == $nonce)
```

Reply:
We utilize a private pipeline in our main repository, which complains about the `$nonce` here as well:

```
src/OidcJwtHelper.php:105 PhanPossiblyUndeclaredVariable Variable $nonce is possibly undeclared
```

See my other suggestion on line 97 how I think we can best fix this.